### PR TITLE
Support to Django 2.0

### PR DESCRIPTION
**Pull request by bertonha**
Add support for Django 2.0 but that makes only valid to Django 1.10 onwards

### CODE REVIEW

File: vinaigrette/models.py [hunk 1]
```diff
@@ -1 +0,0 @@
-# Empty file that nonetheless makes Django happy
```
Comment (jordiecometrica) [inline vinaigrette/models.py]: Why remove the comment?

File: vinaigrette/__init__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 from django.db.models.signals import pre_save, post_save
 from django.conf import settings
-from django.core.urlresolvers import reverse
+from django.urls import reverse
```
Comment (jordiecometrica) [inline vinaigrette/__init__.py]: It appears this is the only functional change?

File: vinaigrette/models.py (hunk 1 above)
Comment (bertonha) [reply vinaigrette/models.py]: I remove the file new versions of djando don't  need to have a models.py

File: vinaigrette/__init__.py (hunk 2 above)
Comment (bertonha) [reply vinaigrette/__init__.py]: On Django 2.0 there no more `django.corr.urlrrsolvers`

**Comment by atodorov** [comment]
Ping, any updates on this one? 

@jordiecometrica I can send a new PR with only the import change if you like ?

**Comment by timo-piirmann** [comment]
+1 for getting this forward.

**Pull request merged**